.github/workflows/sync-client-side-experiments-config-to-cdn.yml

```diff
@@ -0,0 +1,30 @@
+name: Sync configuration to the CDN
```

Comment:
Given that this will be listed with all the other `amphtml` workflows, can we use a more specific name? E.g. "Sync CSE Config to CDN"

![image](https://user-images.githubusercontent.com/26553114/136624557-4743fa7c-92d8-4e1a-b2cb-67d3794044a7.png)